.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/src/MockBot.Api/"
+    schedule:
+      interval: "daily"
+      time: "22:00"
```

Comment:
Similar comment about timezone as here: https://github.com/buerokratt/DMR/pull/64#discussion_r929583036

Reply:
I'm resolving this convo in light of your response on the above thread. Thanks!